docs/support-bundle-guide.md

```diff
@@ -0,0 +1,253 @@
+- [Usage examples](#usage-examples)
+  - [Running antctl commands](#running-antctl-commands)
+  - [Applying SupportBundleCollection CR](#applying-supportbundlecollection-cr)
+- [List of collected items](#list-of-collected-items)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Prerequisites
+
+The `antctl supportbundle` command is supported in Antrea since version 0.7.0.
+
+The `SupportBundleCollection` CRD is introduced in Antrea v1.10.0 as an alpha
+feature. The feature gate must be enabled in both antrea-controller and
+antrea-agent configurations. If you plan to collect support bundle on an external
+Node, you should enable it in the configuration on the external Node as well.
+
+```yaml
+  antrea-agent.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+A single Namespace (e.g., default) is created for saving the Secrets that are
+used to access the support bundle file server, and the permission to read Secrets
+in this Namespace is given to antrea-controller by modifying and applying the
+[RBAC file](/build/yamls/externalnode/support-bundle-collection-rbac.yml).
+
+```yaml
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-read-secrets
+  namespace: default  # Change the Namespace to where the Secret for file server's authentication credential is created.
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: antrea-secret-reader
+subjects:
+  - kind: ServiceAccount
+    name: antrea-controller
+    namespace: kube-system
+```
+
+## The SupportBundleCollection CRD
+
+SupportBundleCollection CRD is introduced to supplement the `antctl` command
+with three additional features:
+
+1. Allow users to collect support bundle files on external Nodes.
+2. Upload all the support bundle files into a user-provided SFTP Server.
+3. Support tracking status of a SupportBundleCollection CR.
+
+Details are provided in the next section.
+
+## Usage examples
+
+### Running antctl commands
+
+Please refer to the [antctl user guide section](antctl.md#collecting-support-information).
+Note: `antctl supportbundle` only works on collecting the support bundles from
+Antrea Controller and Antrea Agent that is running on a K8s Node, but it does
+not work for the Agent on an external Node.
+
+### Applying SupportBundleCollection CR
+
+In this section, we will create two SupportBundleCollection CRs for K8s Nodes
+and external Nodes. Assume we have a cluster with Nodes named "worker1" and
+"worker2". In addition, we have set up two external Nodes named "vm1" and "vm2"
+in the "vm-ns" Namespace by following the instruction in [Install ExternalNode on VM](external-node.md#install-antrea-agent-on-vm).
```

Comment:
`in [Install ExternalNode on VM]` -> `of the [VM installation guide]`

Reply:
updated.